lib/cocaine.rb

```diff
@@ -3,6 +3,8 @@
 require 'rbconfig'
 require 'cocaine/os_detector'
 require 'cocaine/command_line'
+require 'cocaine/command_line/output'
+require 'cocaine/command_line/multi_pipe'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.